source/base/ByteBuffer.ooc

```diff
@@ -28,26 +28,29 @@ ByteBuffer: class {
 	size ::= this _size
 	_referenceCount: ReferenceCounter
 	referenceCount ::= this _referenceCount
-	init: func (=_pointer, =_size) {
+	_owner: Bool
```

Comment:
It's used to know if the ByteBuffer is responsible of freeing the allocated memory on `free`